### DISCUSSION TITLE
无法授权专业版给其他设备

### DISCUSSION THREAD

**Discussion by xelmirage** [getlantern/lantern-forum-cn#1052] [General]
通过授权码授权其他设备时，弹框报错“我们遇到了技术问题”，也无法收到邮件所以无法用邮件授权，现在只有一个终端能试用专业版，麻烦看一下，谢谢

@Mike777Khoo

**Comment by MaxMKMax** [comment]
@xelmirage 您好, 请您提供我支付单据和账号邮箱. 我帮您提交去处理. 谢谢您.

**Reply by xelmirage** [reply]
![2b3cf9c00b4c3f85b35358069ff15e6](https://github.com/user-attachments/assets/25f09744-ce7e-4d47-90f6-c03f8c8122bf)


xelmirage（at）outlook.com


十分感谢！
